source/plot/LinePlotData2D.ooc

```diff
@@ -2,22 +2,35 @@ use ooc-collections
 use ooc-math
 use ooc-draw
 import PlotData2D
+import svg/Shapes
+
+LineStyle: enum {
+	Solid
+	Dashed
+	Dotted
+}
 
 LinePlotData2D: class extends PlotData2D {
 
-	init: func ~default() {
+	lineStyle: LineStyle { get set }
+
+	init: func ~default(lineStyle := LineStyle Solid) {
+		this lineStyle = lineStyle
 		super()
 	}
 
-	init: func ~dataSeries(dataSeries: VectorList<FloatPoint2D>, label := "", colorBgra := ColorBgra new()) {
+	init: func ~dataSeries(dataSeries: VectorList<FloatPoint2D>, label := "", colorBgra := ColorBgra new(), lineStyle := LineStyle Solid) {
+		this lineStyle = lineStyle
 		super(dataSeries, label, colorBgra)
 	}
 
-	init: func ~color(dataSeries: VectorList<FloatPoint2D>, colorBgra: ColorBgra) {
+	init: func ~color(dataSeries: VectorList<FloatPoint2D>, colorBgra: ColorBgra, lineStyle := LineStyle Solid) {
+		this lineStyle = lineStyle
```

Comment:
In this case, you can call `this init(dataSeries, "", colorBgra, lineStyle)` and skip `super`.